tests/Feature/Api/V1/LeaderboardEntriesTest.php

```diff
@@ -0,0 +1,234 @@
+            ->assertNotFound()
+            ->assertJson([]);
+    }
+
+    public function testGetLeaderboardEntries(): void
+    {
+        Carbon::setTestNow(Carbon::now());
+
+        /** Set up a leaderboard with 5 entries: */
+
+        /** @var System $system */
+        $system = System::factory()->create();
+
+        /** @var Game $game */
+        $game = Game::factory()->create(['ConsoleID' => $system->ID]);
+
+        /** @var Leaderboard $leaderboard */
+        $leaderboard = Leaderboard::factory()->create([
+            'GameID' => $game->ID,
+            'Title' => "Test leaderboard 1",
+            'Description' => "I am the first leaderboard",
+        ]);
+
+        $userOne = User::factory()->create(['User' => 'myUser1']);
+        $leaderboardEntryOne = LeaderboardEntry::factory()->create([
+            'leaderboard_id' => $leaderboard->ID,
+            'user_id' => $userOne->ID,
+            'score' => 1,
+        ]);
+
+        $userTwo = User::factory()->create(['User' => 'myUser2']);
+        $leaderboardEntryTwo = LeaderboardEntry::factory()->create([
+            'leaderboard_id' => $leaderboard->ID,
+            'user_id' => $userTwo->ID,
+            'score' => 1,
+        ]);
+
+        $userThree = User::factory()->create(['User' => 'myUser3']);
+        $leaderboardEntryThree = LeaderboardEntry::factory()->create([
+            'leaderboard_id' => $leaderboard->ID,
+            'user_id' => $userThree->ID,
+            'score' => 100,
+        ]);
+
+        $userFour = User::factory()->create(['User' => 'myUser4']);
+        $leaderboardEntryFour = LeaderboardEntry::factory()->create([
+            'leaderboard_id' => $leaderboard->ID,
+            'user_id' => $userFour->ID,
+            'score' => 300,
+        ]);
+
+        $userFive = User::factory()->create(['User' => 'myUser5']);
+        $leaderboardEntryFive = LeaderboardEntry::factory()->create([
+            'leaderboard_id' => $leaderboard->ID,
+            'user_id' => $userFive->ID,
+            'score' => 200,
+        ]);
+
+        /** @var Leaderboard $timedLeaderboard */
+        $timedLeaderboard = Leaderboard::factory()->create([
+            'GameID' => $game->ID,
+            'Title' => "Test leaderboard 2",
+            'Description' => "I am a timed leaderboard",
+            'Format' => "TIME",
+        ]);
+
+        $timedLeaderboardEntryOne = LeaderboardEntry::factory()->create([
+            'leaderboard_id' => $timedLeaderboard->ID,
+            'user_id' => $userOne->ID,
+            'score' => 123,
+        ]);
+
+        $this->get($this->apiUrl('GetLeaderboardEntries', ['i' => $leaderboard->ID]))
+            ->assertSuccessful()
+            ->assertJson([
+                'Count' => 5,
+                'Total' => 5,
+                'Results' => [
+                    [
+                        "Rank" => $leaderboard->getRank($leaderboardEntryFour->score),
```

Comment:
Recommend hardcoding all of these ranks in the tests.

If the implementation details of `$leaderboard->getRank()` change, the tests will be invalid but won't fail.